.travis/trigger-release.sh

```diff
@@ -10,7 +10,7 @@ GITHUB_REPOSITORY_NAME=atlas-checks
 # - The travis client is installed: gem install travis
 # - travis login --org has been called to authenticate
 
-TRAVIS_PERSONAL_TOKEN=$(travis token)
+TRAVIS_PERSONAL_TOKEN=$(d4f235d37440307220eebc7dfb2b38f09d06a461)
```

Comment:
please revert this!

Reply:
Yikes!  I reverted it.  Hopefully this is gone once we squash the commits.
@danielduhh

Reply:
Its okay - you can regenerate your API key 😄